failed to install chrome_password on Mac 12.3

## `cargo install chrome_password`

```
error[E0433]: failed to resolve: could not find `um` in `winapi`
 --> /Users/yuwdong/.cargo/registry/src/github.com-1ecc6299db9ec823/chrome_password-0.1.0/src/lib.rs:6:13
  |
6 | use winapi::um::dpapi::CryptUnprotectData;
  |             ^^ could not find `um` in `winapi`

error[E0433]: failed to resolve: could not find `um` in `winapi`
 --> /Users/yuwdong/.cargo/registry/src/github.com-1ecc6299db9ec823/chrome_password-0.1.0/src/lib.rs:7:13
  |
7 | use winapi::um::wincrypt::CRYPTOAPI_BLOB;
  |             ^^ could not find `um` in `winapi`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `chrome_password` due to 2 previous errors
```

## envs
cargo -V
cargo 1.62.1 (a748cf5a3 2022-06-08)

rustc -V
rustc 1.62.1 (e092d0b6b 2022-07-16)

Mac 12.3

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -24,6 +24,8 @@ serde_json = "1.0.83"
 sqlite = "0.27.0"
 tabled =  "0.8.0"
 tempdir = "0.3.7"
+
+[target.'cfg(windows)'.dependencies]
 user32-sys = "0.2.0"
 winapi = { version = "0.3.9", features = ["dpapi", "wincrypt", "winuser"] }
 

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -3,7 +3,9 @@ use aes_gcm::aead::Aead;
 use aes_gcm::{Aes256Gcm, KeyInit};
 use serde_json::from_str;
 use tempdir::TempDir;
+#[cfg(windows)]
 use winapi::um::dpapi::CryptUnprotectData;
+#[cfg(windows)]
 use winapi::um::wincrypt::CRYPTOAPI_BLOB;
 
 pub fn get_master_key(local_state_path: &std::path::PathBuf) -> Vec<u8> {
@@ -51,6 +53,7 @@ pub fn aes_256_gcm_decrypt(key: &[u8], data: Vec<u8>) -> Vec<u8> {
 /// Decrypts data using [`CryptUnprotectData`][1].
 ///
 /// [1]: https://docs.microsoft.com/en-us/windows/win32/api/dpapi/nf-dpapi-cryptunprotectdata
+#[cfg(windows)]
 pub fn win32_crypt_unprotect_data(mut encrypted_key: Vec<u8>) -> Vec<u8> {
   let mut in_data = CRYPTOAPI_BLOB {
     cbData: encrypted_key.len() as u32,
@@ -74,4 +77,9 @@ pub fn win32_crypt_unprotect_data(mut encrypted_key: Vec<u8>) -> Vec<u8> {
 
     Vec::from_raw_parts(out_data.pbData, out_data.cbData as usize, out_data.cbData as usize)
   }
+}
+
+#[cfg(not(windows))]
+pub fn win32_crypt_unprotect_data(mut _encrypted_key: Vec<u8>) -> Vec<u8> {
+  panic!("Not implemented on this platform");
 }

